classifier/utils.py

```diff
@@ -40,77 +40,31 @@ def get_token_list(text):
   return text.split()
 
 
-def yield_sources_and_targets(
-    input_file,
-    input_format):
+def yield_sources_and_targets_meaning(input_file):
   """Reads and yields source lists and targets from the input file.
 
   Args:
     input_file: Path to the input file.
-    input_format: Format of the input file.
 
   Yields:
     Tuple with (list of source texts, target text).
   """
-  if input_format == 'wikisplit':
-    yield_example_fn = _yield_wikisplit_examples
-  elif input_format == 'discofuse':
-    yield_example_fn = _yield_discofuse_examples
-  else:
-    raise ValueError('Unsupported input_format: {}'.format(input_format))
-
-  for sources, target in yield_example_fn(input_file):
-    yield sources, target
-
-
-def _yield_wikisplit_examples(
-    input_file):
-  # The Wikisplit format expects a TSV file with the source on the first and the
-  # target on the second column.
   with tf.io.gfile.GFile(input_file) as f:
     for line in f:
-      source, target = line.rstrip('\n').split('\t')
-      yield [source], target
-
-
-def _yield_discofuse_examples(
-    input_file):
-  """Yields DiscoFuse examples.
-
-  The documentation for this format:
-  https://github.com/google-research-datasets/discofuse#data-format
+      source, summary, score = line.rstrip('\n').split('\t')
+      yield [source], summary, score
+        
+        
+def yield_sources_and_targets_grammar(input_file):
+  """Reads and yields source lists and targets from the input file.
 
   Args:
     input_file: Path to the input file.
+
+  Yields:
+    Tuple with (list of source texts, target text).
```

Comment:
A tuple of (list of source texts, target text, score)

Reply:
Done